package.json

```diff
@@ -85,34 +86,31 @@
       "build:open": "cd ./Open && nuxt build",
       "build:api": "cross-env NODE_ENV=production tsc --project Server/tsconfig.json",
       "build:discord-bot": "cross-env NODE_ENV=production tsc --project DiscordBot/tsconfig.json",
-
```

Comment:
readd spaces cuz package.json removes these when installing a new package for w/e reason